drivers/docker/handle.go

```diff
@@ -332,6 +335,8 @@ func (h *taskHandle) run() {
 
 	// Stop the container just incase the docker daemon's wait returned
 	// incorrectly.
```

Comment:
Maybe it's just because I haven't touched this code in ages, but it took me some digging to determine why it was ok to ignore kill_timeout here. A bit of a `why` comment could help here, but nbd either way.

```suggestion
	// incorrectly. Container should have exited by now so kill_timeout can be ignored.
```